### PR TITLE
remove teacher_resources from all script json files

### PR DESCRIPTION
**Pull request by davidsbailey**
Continues [PLAT-1680](https://codedotorg.atlassian.net/browse/PLAT-1680). See [work plan](https://docs.google.com/document/d/1yIL2Z7aqFTdbCi24-RpUhPO_B3wRdsiKRTB1EEs17EY/edit#heading=h.1l25aiwflbrc).

To make this change, I ran:
```
[development] dashboard > scripts = Script.all.select(&:teacher_resources)
[development] dashboard > scripts.count
=> 53

[development] dashboard > scripts.each do |s|
  s.update!(teacher_resources: nil)
  s.write_script_json
end
```
and then manually reverted changes to irrelevant lines including the serialized_at timestamp.

## Testing story

just double checked that teacher resources till appear locally on coursea-2020.

### CODE REVIEW

File: dashboard/config/scripts_json/vpl-csd-2022.script_json [hunk 1]
```diff
@@ -8,20 +8,6 @@
       "is_course": true,
       "is_migrated": true,
       "student_detail_progress_view": true,
-      "teacher_resources": [
```
Comment (bethanyaconnor) [inline dashboard/config/scripts_json/vpl-csd-2022.script_json:11]: These must've been added recently. Is anything going to break for these PL scripts?

**Review by bethanyaconnor — APPROVED** [review]
Question about PL courses but otherwise LGTM! Thanks for cleaning this up

File: dashboard/config/scripts_json/vpl-csd-2022.script_json (hunk 1 above)
Comment (dmcavoy) [reply dashboard/config/scripts_json/vpl-csd-2022.script_json:11]: The following lines make be think these were a result of copying last years and have not been updated yet. VPL is used during academic year workshops so I'm guessing it's not finalized yet. I think its fine to make this change
```
        [
          "curriculum",
          "https://studio.code.org/courses/csd-2021"
        ],
```

**Review by dmcavoy — APPROVED** [review]

**Comment by davidsbailey** [comment]
Thanks Bethany and Dani! I will plan to wait til Monday to merge this, partially to avoid deploying risky things on Friday, but also to give a short period of time to see if we get any zendesk tickets about missing translations after https://github.com/code-dot-org/code-dot-org/pull/46638 ships.

**Pull request merged**